scala-api/src/main/scala/com/cloudera/livy/scalaapi/ScalaJobHandle.scala

```diff
@@ -89,10 +158,37 @@ class ScalaJobHandle[T] private[livy] (jobHandle: JobHandle[T]) extends Future[T
     }
   }
 
+  /**
+   * Await the completion of the job and return the result (of type `T`)
+   *
+   * Although this method is blocking, the internal use of [[scala.concurrent.blocking blocking]]
```

Comment:
Does this apply to this implementation at all?

You should probably just say this is here to support Scala's `Await`, you don't need to go into implementation details here since no one will call this directly.